algorithms/CPlusPlus/Arrays/vector.cpp

```diff
@@ -0,0 +1,216 @@
+#include <iostream>
+#include <cmath>
+#include <cassert>
+using namespace std ;
+
+class Vector{
+
+private:
+
+	int *arr = nullptr ;
+	int size , capacity ;
+
+	void expand_capaxity(){
```

Comment:
Is there any reason to misspelling `expand_capaxity` ?

Reply:
Should be `expand_capacity`?